Components/Cards/cards.css

```diff
@@ -0,0 +1,209 @@
+/* cards  */
+
+.card {
+  background-color: var(--light-color);
+  min-width: 20rem;
+  max-width: 20rem;
+  max-height: 31.25rem;
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.5rem;
+  margin: 0.5rem;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+  position: relative;
+  transition: all 0.5s ease;
+}
+.card:hover {
+  cursor: pointer;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px,
+    rgba(0, 0, 0, 0.22) 0px 10px 10px;
+}
+
+/* overlay on card */
+.overlay-content {
```

Comment:
You can use shorthand property for position as below
position: absolute 10px 50px 50px 100px;

Reply:
Oh. Thanks. I will make this changes.